Example/Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -15,6 +15,7 @@
 			dependencies = (
 			);
 			name = "boost-for-react-native";
+			productName = "boost-for-react-native";
```

Comment:
This line keeps getting added, I had checked with Orta in the past if it was an issue and he gave it the green light that it shouldn't effect anything.